src/cz/crcs/ectester/standalone/ECTesterStandalone.java

```diff
@@ -194,8 +195,8 @@ private TreeCommandLine parseArgs(String[] args) throws ParseException {
         testOpts.addOption(Option.builder("st").longOpt("sig-type").desc("Set the Signature object [type].").hasArg().argName("type").optionalArg(false).build());
         testOpts.addOption(Option.builder("f").longOpt("format").desc("Set the output format, one of text,yaml,xml.").hasArg().argName("format").optionalArg(false).build());
         testOpts.addOption(Option.builder().longOpt("key-type").desc("Set the key [algorithm] for which the key should be derived in KeyAgreements with KDF. Default is \"AES\".").hasArg().argName("algorithm").optionalArg(false).build());
+        testOpts.addOption(Option.builder("s").longOpt("suite").desc("Set the test suite: <suite>").hasArg().argName("suite").optionalArg(false).build());
```

Comment:
This shouldn't be added, the test-suite is passed as an argument and not an option. See the help text for the ECTesterStandalone.jar without this PR merged, there is a `test-suite` argument. See line 198 of the file. I think you access it by using `cli.getArg(0)` or something like that, I don't remember exactly, but the library choice is handled similarly with the practice that the library is always the last argument (So test-suite will be the first in this case).

Reply:
This should now be resolved. I made this change when I first begun working with ECTester, and wasn't aware of this practice at that time.